packages/SwingSet/misc-tools/monitor-slog-block-time.py

```diff
@@ -3,6 +3,9 @@
 from collections import defaultdict
 import subprocess
 
+print("writing to monitor.out")
+mf = open("monitor.out", "a", buffering=1)
+
 # run like this:
 #   tail -n 10000 -F chain.slog | python3 monitor-slog-block-time.py [keys.txt]
```

Comment:
Is **keys.txt** a well-known name? If not, it seems like we actually want **keys.json**.
```suggestion
#   tail -n 10000 -F chain.slog | python3 monitor-slog-block-time.py [keys.json]
```